modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationNode.java

```diff
@@ -104,22 +101,24 @@ protected ConfigurationNode(
     /** {@inheritDoc} */
     @Override
     public void listen(ConfigurationListener<VIEWT> listener) {
-        updateListeners.add(listener);
+        updateListeners.addListener(listener, changer.notificationCount() + 1);
```

Comment:
I feel like this statement breaks encapsulation. Should `+ 1` be inside of the method, not outside?

Reply:
Fix it.